code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -388,9 +388,11 @@
 		msg += "<span class = 'deptradio'>Physical status:</span> <a href='?src=[UID()];medical=1'>\[[medical]\]</a>\n"
 		msg += "<span class = 'deptradio'>Medical records:</span> <a href='?src=[UID()];medrecord=`'>\[View\]</a> <a href='?src=[UID()];medrecordadd=`'>\[Add comment\]</a>\n"
 
-
 	if(print_flavor_text() && !skipface)
-		msg += "[print_flavor_text()]\n"
+		if(get_organ("head"))
+			var/obj/item/organ/external/head/H = get_organ("head")
+			if(!(H.disfigured || cloneloss > 50 || (HUSK in mutations)))
```

Comment:
According to what I see in the code, if someone's husked, they would be automatically disfigured. 

Is there a situation where: 
- Someone is mutilated solely due to clone loss, yet somehow not disfigured
- Someone is husked yet somehow not disfigured? 

I think disfigured should be the only check here, else people's flavour text could potentially disappear after plastic surgery but before their clone damage or husk mutations is cured (Not sure if there's an edgecase like that)

Reply:
You're absolutely correct on the removal of the husk check, there should not be a case in which someone's husked without their head being disfigured.

About the clone loss, this is solely taken from mob/living/carbon/human/human.dm in which someone's identity is returned as unknown if clone loss is over 50 (I don't exactly know why cellular damage would make someone unidentifiable, I'm just trying to be consistent with get_face_name() here). It is very much possible (though rare) to cause over 50 cloneloss without being disfigured eg. by using the decloner on them.

```//Returns "Unknown" if facially disfigured and real_name if not. Useful for setting name when polyacided or when updating a human's name variable
/mob/living/carbon/human/proc/get_face_name()
	var/obj/item/organ/external/head = get_organ("head")
	if(!head || head.disfigured || cloneloss > 50 || !real_name || (HUSK in mutations))	//disfigured. use id-name if possible
		return "Unknown"
	return real_name
```

Reply:
Make sense to me. Can you change it to just check for disfigured?

I'd also request that the cloneloss check be removed from get face name, but that should be a separate PR as I am not familiar with the history behind it.